source/Cosmos.System2/Graphics/Canvas.cs

```diff
@@ -452,12 +451,7 @@ protected bool CheckIfModeIsValid(Mode mode)
 
         protected void ThrowIfModeIsNotValid(Mode mode)
         {
-            if (mode == null)
-            {
-                Global.mDebugger.SendInternal($"mode is null raising exception!");
-                throw new ArgumentNullException(nameof(mode));
-            }
-
+            /* Mode has been changed to be a structure so it cannot be null anymore */
```

Comment:
Nit: I think that this comment doesn't make it easier to understand the code, and may create some confusion too.